src/commands/block/get.ts

```diff
@@ -14,13 +14,33 @@
  *
  */
 import BaseCommand from '../../base';
-import getAPIClient from '../../utils/api';
+import { getAPIClient } from '../../utils/api';
 import { query } from '../../utils/query';
 
 export default class GetCommand extends BaseCommand {
-	async run() {
-		const { args: { blockIds } } = this.parse(GetCommand);
-		const req = blockIds.map(blockId => ({
+	static args = [
+		{
+			name: 'blockIds',
+			required: true,
+			description: 'Comma-separated block ID(s) to get information about.',
+		},
+	];
+
+	static description = 'Gets block information from the blockchain.';
+
+	static examples = [
+		'block:get 17108498772892203620',
+		'block:get 17108498772892203620,8541428004955961162',
+	];
+
+	static flags = {
+		...BaseCommand.flags,
+	};
+
+	async run(): Promise<void> {
+		const { args: { blockIds: blockIdsStr } } = this.parse(GetCommand);
+		const blockIds = blockIdsStr.split(',').filter(Boolean);
```

Comment:
can't we use `parse: input => input.split(',').filter(Boolean)` in args?

Reply:
it seems like type doesn't match to the `oclif` definition.
I guess originally it was wrong usage ='(